geode-core/src/main/java/org/apache/geode/management/internal/api/GeodeClusterManagementServiceFactory.java

```diff
@@ -74,45 +77,38 @@ public ClusterManagementService create() throws IllegalStateException {
       Set<InternalDistributedMember> locatorsWithClusterConfig =
           cacheImpl.getDistributionManager().getAllHostedLocatorsWithSharedConfiguration()
               .keySet();
-      String serviceAddress = getHttpServiceAddress(locatorsWithClusterConfig);
 
-      return new ClientClusterManagementService(serviceAddress);
+      MemberInformation memberInformation = getHttpServiceAddress(locatorsWithClusterConfig);
+
+      return create(getHostName(memberInformation), memberInformation.getHttpServicePort(),
```

Comment:
At this point you need to ensure you're picking up and using the SSL options that have been set on this server.